.github/workflows/link-repair.yml

```
@@ -44,8 +44,8 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           REPO_OWNER: ${{ github.repository_owner }}
           REPO_NAME: ${{ github.event.repository.name }}
-          DRY_RUN: ${{ github.event.inputs.dry_run || 'false' }}
-          MIN_CONFIDENCE: ${{ github.event.inputs.min_confidence || '0.6' }}
+          DRY_RUN: ${{ coalesce(github.event.inputs.dry_run, 'false') }}
```

actionlint (1.7.12)

[error] undefined function "coalesce". available functions are "always", "cancelled", "case", "contains", "endswith", "failure", "format", "fromjson", "hashfiles", "join", "startswith", "success", "tojson"

(expression)